data-assets/acm_right_sizing_grafana_dashboard.yaml

```diff
@@ -924,7 +924,7 @@ data:
           "targets": [
             {
               "exemplar": true,
-              "expr": "acm_rs:namespace:memory_usage{cluster=\"$cluster\", profile=\"$profile\"}",
+              "expr": "topk(${top_n},\nsum by (namespace) (acm_rs:namespace:memory_usage{cluster=\"$cluster\", profile=\"$profile\"})\n/\nsum by (namespace) (acm_rs:namespace:memory_request{cluster=\"$cluster\", profile=\"$profile\"})\n)",
```

Comment:
Let's not give option to user to choose top_n value, by default you can keep it to 20 for now.

Reply:
Done.